### ISSUE TITLE
Unable to reorder mail accounts

### ISSUE THREAD

**Issue by mslourens** [Foundry376/Mailspring#1577]
When dragging the mail accounts in the accounts preferences page, the accounts are not reordering and I get this error in the Developer tools:

```
Each child in an array or iterator should have a unique "key" prop.

Check the render method of 'EditableList'
```

**Comment by bengotow** [comment]
Oh man good catch @mslourens! This must have been a regression at some point—will get this fixed!

**Comment by bengotow** [comment]
Hey folks! This shipped today in 1.7.2 - you can download the update from our website at getmailspring.com/download or choose "Check for Updates" in the app's menu. Enjoy the update!
